lib/std/Build/Step/ConfigHeader.zig

```diff
@@ -181,17 +184,26 @@ fn make(step: *Step, prog_node: *std.Progress.Node) !void {
     man.hash.add(@as(u32, 0xdef08d23));
     man.hash.addBytes(self.include_path);
     man.hash.addOptionalBytes(self.include_guard_override);
+    man.hash.addOptionalBytes(self.auto_generated_comment);
 
     var output = std.ArrayList(u8).init(gpa);
     defer output.deinit();
 
-    const header_text = "This file was generated by ConfigHeader using the Zig Build System.";
-    const c_generated_line = "/* " ++ header_text ++ " */\n";
-    const asm_generated_line = "; " ++ header_text ++ "\n";
+    const default_header_text = "This file was generated by ConfigHeader using the Zig Build System.";
+    const c_generated_line = "/* " ++ default_header_text ++ " */\n";
+    const asm_generated_line = "; " ++ default_header_text ++ "\n";
+
+    if (self.auto_generated_comment) |comment| {
+        try output.appendSlice(comment);
+    } else {
+        switch (self.style) {
+            .autoconf, .cmake, .blank => try output.appendSlice(c_generated_line),
+            .nasm => output.appendSlice(asm_generated_line),
```

Comment:
Shouldn't this line have the "try" as well?

Reply:
https://github.com/ziglang/zig/pull/20018
Corrected